reactiongamino.ino

```diff
@@ -183,28 +184,29 @@ static void Game_SuccessfullyPressedButtonStateHandler(Game_InstanceData *inst_d
         lcd.print("Barely made it");
       }
 
-      inst_data->reaction_time += inst_data->time_counter;
+      inst_data->reaction_time += inst_data->time_counter + 250; //250 is LED on delay
       ++inst_data->successful_hits;
 
 
-      if (instance_data.loop_delay >= 25)
+      if (instance_data.loop_delay >= 5)
       {
-        instance_data.loop_delay -= 25;
+        instance_data.loop_delay -= (instance_data.loop_delay / 20);
       }
-      if (inst_data->timeout >= 25)
+      if (inst_data->timeout >= 100)
       {
         inst_data->timeout -= 5;
       }
       else
       {
-        inst_data->timeout = 5;
+        inst_data->timeout = 25;

```

Comment:
This was actually a bug fix.
we had a '-5 to timeout if above 25' but if it was below 25 we set it to 5 making players instant fail.